apps/site/src/app/(public)/page.tsx

```diff
@@ -1,9 +1,9 @@
+import React, { Fragment } from "react"
+import type { Metadata } from "next"
 import { find } from "@/lib/strapi"
 import { StrapiSeoFormate } from "@/lib/strapiSeo"
-import type { Metadata } from "next"
-import DynamicBlockRenderer from "./body"
-import React from "react"
 import { getLanguageFromCookie } from "@/utils/language"
+import { getPublicComponents } from "@padma/metajob-ui"
```

Comment:
### Issue: Hardcoded Import

 **Description**
We cannot import like this: `import { MyProfile } from "@padma/metajob-ui"`.  
We need to ensure that any theme can be used. Avoid hardcoding `@padma/metajob-ui`.

Reply:
This import will be updated in the next updated PR